hack/lib/golang.sh

```diff
@@ -677,9 +677,7 @@ kube::golang::build_binaries() {
     local host_platform
     host_platform=$(kube::golang::host_platform)
 
-    # Use eval to preserve embedded quoted strings.
     local goflags goldflags goasmflags gogcflags
```

Comment:
we can also probably remove the `goflags` local variable and then remove all references to `$goflags`?

Reply:
oh, scratch that